fastlane/Fastfile

```diff
@@ -11,6 +11,102 @@ before_all do
     setup_ci()
   end
 end
+desc "Release a new version"
+lane :release do |options|
+  UI.user_error!("Please use type parameter with one of the options: type:patch, type:minor, type:major") unless ["patch", "minor", "major"].include?(options[:type])
+  
+  ensure_git_branch(branch: 'main') # We can only release on default branch
+  ensure_git_status_clean unless options[:no_ensure_clean]
+  
+  version_number = increment_version_number_in_plist(bump_type: options[:type], xcodeproj: "StreamChat.xcodeproj", target: "StreamChat")
+
+  set_SDK_version(version: version_number)
+
+  increment_version_number_in_plist(version_number: version_number, xcodeproj: "StreamChat.xcodeproj", target: "StreamChat")
```

Comment:
You don't need this, you already do that in line 21

Reply:
Thanks, fixed.